extensions/amp-video/1.0/video-wrapper.js

```diff
@@ -106,12 +106,8 @@ export function VideoWrapper({
 
   useLayoutEffect(() => {
     if (mediasession && playing && metadata) {
-      setMediaSession(window, metadata, play, pause);
+      return setMediaSession(window, metadata, play, pause);
```

Comment:
This is a lot better! Very easy to get used to the symmetrical effect/undo style.

Reply:
It's quite nice, I agree!